ClosedXML/Excel/Drawings/XLPicture.cs

```diff
@@ -436,13 +438,13 @@ private static string GetMimeType(Image i)
             return "image/unknown";
         }
 
-        private void DeduceDimensionsFromBitmap(Bitmap bitmap)
+        private void DeduceDimensionsFromBitmap(Image image)
         {
-            this.OriginalWidth = bitmap.Width;
-            this.OriginalHeight = bitmap.Height;
+            this.OriginalWidth = image.Width;
+            this.OriginalHeight = image.Height;
 
-            this.width = bitmap.Width;
-            this.height = bitmap.Height;
+            this.width = image.Width;
+            this.height = image.Height;
```

Comment:
Can we also rename private fields to match the convention? Like, `_width`, `_height`, `_id`, etc.?

Reply:
If I had my choice from the start, I would choose `width_`, etc. It helps with the autocompletion in Visual Studio. I hate having to type the `_`. But yes, I will do this one that you suggest.

Reply:
Should
`private const String InvalidNameChars = @":\/?*[]";`
become
`private const String _invalidNameChars = @":\/?*[]";` ?

Reply:
Microsoft's convention is `InvalidNameChars`

https://stackoverflow.com/questions/242534/c-sharp-naming-convention-for-constants

Reply:
I was toooo slooowww :)